who-europe-harvard.csl

```diff
@@ -52,8 +52,8 @@
         <group delimiter=", ">
           <text variable="URL"/>
           <group>
-            <text value="accessed "/>
-            <date variable="accessed ">
+            <text value="accessed"/>
+            <date variable="accessed">
```

Comment:
This would certainly be more general. Moreover, it looks like the space after the text "accessed" should stay, but I will move it to a group delimiter. I will change that.